site/app/templates/Notifications.twig

```diff
@@ -0,0 +1,50 @@
+<div class="content">
+    <div style="float: right; margin-bottom: 20px;">
+         <a href="{{ core.buildUrl({"component": "navigation", "page": "notifications", "action": "mark_all_as_seen"}) }}" class="btn btn-primary">Mark all as seen</a>
+        {% if show_all %}
+            <a href="{{ core.buildUrl({"component": "navigation", "page": "notifications"}) }}" class="btn btn-primary">Show unread only</a>
+        {% else %}
+            <a href="{{ core.buildUrl({"component": "navigation", "page": "notifications", "show_all": "1"}) }}" class="btn btn-primary">Show All</a>
+        {% endif %}
+    </div>
+    <h2>Notifications</h2>
+    <table class="table table-bordered persist-area">
+                <tr class="info persist-header">
+                    <td colspan="8" style="text-align: center">Course: {{ course  }}</td>
```

Comment:
It might be better to have "New notifications" and "All notifications" for the table since when a user is currently viewing the notifications page it is within a course.